### ISSUE TITLE
Unable to add machine, db locked

### ISSUE THREAD

**Issue by jdinkel** [conjure-up/conjure-up#886]
## Report

Thank you for trying **conjure-up**! Before reporting a bug please make sure you've gone through this checklist:

- [x] Is this problem already documented at https://docs.ubuntu.com/conjure-up/en/troubleshoot#common-spell-problems **no**
- [x] Is conjure-up running inside a virtual machine? **yes, kvm**
- [x] Is this problem reproducible with `sudo snap refresh conjure-up --edge`? **yes**

## Please provide the output of the following commands

```
which juju: /snap/bin/juju
which conjure-up: /snap/bin/conjure-up
which lxd: /usr/bin/lxd
juju version: 2.1.2-xenial-amd64
[conjure-up.tar.gz](https://github.com/conjure-up/conjure-up/files/999020/conjure-up.tar.gz)

conjure-up --version: conjure-up 2.1.5
lxd --version: 2.13
lxc config show

> config:
>   core.https_address: '[::]'

cat /etc/lsb-release

> DISTRIB_ID=Ubuntu
> DISTRIB_RELEASE=16.04
> DISTRIB_CODENAME=xenial
> DISTRIB_DESCRIPTION="Ubuntu 16.04.2 LTS"
```

Please attach tarball of **~/.cache/conjure-up**:

```
tar cvzf conjure-up.tar.gz ~/.cache/conjure-up
```

## Sosreport

Please attach a sosreport:

```
sudo apt install sosreport
sosreport

> sosreport (version 3.4)
> 
> no valid plugins were enabled

```

The resulting output file can be attached to this issue.

## What Spell was Selected?

OpenStack with NovaLXD

## What provider (aws, maas, localhost, etc)?

localhost

## Commands ran

Please outline what commands were run to install and execute conjure-up:

On a fresh install of Ubuntu 16.04.2

    sudo snap install conjure-up --classic
    conjure-up

**Comment by jdinkel** [comment]
[conjure-up.tar.gz](https://github.com/conjure-up/conjure-up/files/999023/conjure-up.tar.gz)

**Comment by battlemidget** [comment]
Hmm this is a new error I've never seen before:

```
2017-05-13 19:05:46,352 [DEBUG] log.py:14 - conjure-up/_unspecified_spell: AddMachines returned {'machines': [{'machine': '', 'error': {'message': 'cannot add a new machine: DB is locked', 'code': ''}}]}
```

In your VM, can you run a `df -h`, Also the fact that sosreport doesn't run is also odd, could you attach your `/var/log/syslog`

**Comment by jdinkel** [comment]
Sorry I already wiped out the install.  It was a fresh install of Ubuntu
Server 16.04.2 64-bit, only updated/full-upgrade and then installed and ran
conjure-up.  And then just 'sudo apt install sosreport' for the bug
report.  However, I reinstalled on the exact same virtual machine with
Ubuntu Desktop 16.04.2 64-bit.  It made it further in the installer, it got
to the list of services and the whole list said active with green
checkmarks, but then left it for a day and it still said something like
waiting for services to start, at the bottom, so there may have been more
services cut of at the bottom of the list that couldn't install - separate
issue anyway.

Anyway, my point is that I'm thinking maybe something gets installed with
the "desktop" version of Ubuntu that does not get installed with the
"server" version and is needed by conjure.

I'll reinstall on the server version and get you the syslog, but I wanted
to drop you this note because my day job is keeping me busy so I may be
slow at it.
